### PR TITLE
chore: add metrics for 360 image add/enter/transition/exit

### PR DESCRIPTION
**Pull request by christjt**
#### Type of change

Chore 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/REV-764

### CODE REVIEW

File: viewer/packages/api/src/public/migration/Cognite3DViewer.ts [hunk 1]
```diff
@@ -736,6 +736,13 @@ export class Cognite3DViewer {
       preMultipliedRotation
     );
 
+    const numberOf360Images = image360Collection.image360Entities.length;
+
+    MetricsLogger.trackEvent('360ImageCollectionAdded', {
```
Comment (pramodcog) [inline viewer/packages/api/src/public/migration/Cognite3DViewer.ts:741]: If `numberOf360Images` is `0`, should we track the event as `360ImageCollectionAdded`

File: viewer/packages/api/src/public/migration/Cognite3DViewer.ts [hunk 2]
```diff
@@ -736,6 +736,13 @@ export class Cognite3DViewer {
...
+      datasource,
+      numberOf360Images
+    });
```
Comment (pramodcog) [inline viewer/packages/api/src/public/migration/Cognite3DViewer.ts:744]: If `numberOf360Images` is `0`, should we track the event as `360ImageCollectionAdded`

Comment (christjt) [reply viewer/packages/api/src/public/migration/Cognite3DViewer.ts:744]: sure, why not? Gives us a good metric on how often people send in and invalid filter, which is good to know.

Comment (pramodcog) [reply viewer/packages/api/src/public/migration/Cognite3DViewer.ts:744]: 👍

**Review by pramodcog — APPROVED** [review]
LGTM

**Pull request merged**